modules/bazel_skylib/1.1.1/MODULE.bazel

```diff
@@ -0,0 +1,11 @@
+module(
+    name = "bazel_skylib",
+    version = "1.1.1",
+    compatibility_level = 1,
```

Comment:
@meteorcloudy is this the correct value for compatibility_level? I've not yet fuly understood the semantics of when this should be bumped.

Reply:
this should be bumped sparingly (e.g. when bazel_skylib 2.x is out).